icicle/appUtils/lde/lde.cuh

```diff
@@ -6,61 +6,57 @@
 
 /**
  * @namespace lde
- * LDE (stands for low degree extension) contains [NTT](@ref ntt)-based methods for translating between coefficient and
- * evaluation domains of polynomials. It also contains methods for element-wise manipulation of vectors, which is useful
- * for working with polynomials in evaluation domain.
+ * LDE (stands for low degree extension) contains [NTT](@ref ntt)-based methods for translating between coefficient and evaluation domains of polynomials.
+ * It also contains methods for element-wise manipulation of vectors, which is useful for working with polynomials in evaluation domain.
```

Comment:
But they are just general-purpose arithmetic. Wouldn't it be better to move these into some other file?

Reply:
I think you're right...
My initial logic was like this: in arkworks there [is](https://github.com/arkworks-rs/algebra/blob/master/poly/src/evaluations/univariate/mod.rs) this `Evaluations` struct which implements all the operations one might perform on polynomials in evaluation form, including element-wise add, sub and mult and I wanted to do something similar in lde. But we might also need vector operations elsewhere (e.g. in tree builders right?) so it's best to factor these out into a more general place
I think I'm going to remove lde for now then, and will (try to) implement a higher-level polynomial functionality in the next PR. Would that make sense?